CHANGELOG.md

```diff
@@ -18,7 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ability to create a custom extractors for unsupported media types
 - Added in PDF extractor
 - Added in a command line model manager tester
-- Pascal VOC format support
+- Ability to dump/upload an annotation in several formats(CVAT, Pascal VOC)
```

Comment:
Ability to dump/load annotations in several formats from UI (CVAT, Pascal VOC, YOLO)